source/core/src/main/com/deco2800/game/physics/PhysicsContactListener.java

```diff
@@ -41,6 +41,8 @@ public void postSolve(Contact contact, ContactImpulse impulse) {
 
   private void triggerEventOn(Fixture fixture, String evt, Fixture otherFixture) {
     BodyUserData userData = (BodyUserData) fixture.getBody().getUserData();
+    BodyUserData otherData = (BodyUserData) fixture.getBody().getUserData();
+
     if (userData != null && userData.entity != null) {
```

Comment:
Check for `otherData != null`

Reply:
Yeah, this was just used for testing. Got rid of it already